inspirehep/modules/orcid/utils.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""ORCID utils."""
+
+from __future__ import absolute_import, division, print_function
+
+import re
+
+from orcid import MemberAPI
+
+from inspire_utils.config import load_config
+from inspire_utils.record import get_value
+
+
+RECID_FROM_INSPIRE_URL = re.compile(
+    r"https?://(?:labs\.)?inspirehep\.net/(?:record|literature)/(\d+)",
+    re.IGNORECASE
+)
+
+WORKS_BULK_QUERY_LIMIT = 50
```

Comment:
IMO this will be more helpful if it's part of the configuration, WDYT?

Reply:
Idk, this constant will not change, unless there's a new version of ORCID API in which case we'd probably need to change the logic here too. Is it worth polluting the config file with such a thing? I made a constant for it, to not have a cryptic 50 hanging in the code...

Reply:
Since it's a constant we can leave it as it is! 👍